### PR TITLE
Provider state initialisation phase

### PR DESCRIPTION
**Pull request by lfportal**
Providers may require common data to be setup prior to running any test. E.g. super user, country data etc. The provider state initialisation phase allows any non specific pre-test setup to be executed.

### CODE REVIEW

File: cli/src/test-utils/test-runner.ts [hunk 1]
```diff
@@ -222,6 +225,35 @@ async function executeRequestUnderTest(
...
 }
 
+/**
+ * Execute the state initialisation request.
+ *
+ * @param stateUrl state change URL
+ */
+async function executeStateInitialisation(stateUrl: string): Promise<boolean> {
+  try {
+    TestLogger.mute("\tPerforming state initialisation request");
```
Comment (committedteadrinker) [inline cli/src/test-utils/test-runner.ts]: what does `mute` do?

**Review by committedteadrinker — APPROVED** [review]
Long live 'initialisation' 🇦🇺

File: cli/src/test-utils/test-runner.ts (hunk 1 above)
Comment (lfportal) [reply cli/src/test-utils/test-runner.ts]: prints in a neutral grey colour

**Review by fwouts — APPROVED** [review]
I actually think we should follow international conventions and use 🇺🇸 English in our code... Just like French developers don't write their open source libraries in 🇫🇷 French.

**Review by Kastet — APPROVED** [review]

**Pull request merged**